mmv1/third_party/terraform/tests/resource_firestore_database_update_test.go.erb

```diff
@@ -0,0 +1,83 @@
+<% autogen_exception -%>
+package google
+
+<% unless version == 'ga' -%>
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccFirestoreDatabase_update(t *testing.T) {
```

Comment:
Consider adding a test for create surface.

Reply:
The examples autogenerate tests that handle database creation. It's only necessary to handwrite the update tests.